pkg/projects/Microsoft.NETCore/Microsoft.NETCore.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <PropertyGroup>
-    <Version>5.0.3</Version>
+    <Version>5.0.2</Version>
```

Comment:
Building this locally results in the following nupkgs in the core-setup/pkg/bin/packages directory:

- Microsoft.NETCore.App.1.1.2-servicing-*
- Microsoft.NETCore.UniversalWindowsPlatform.5.3.2-servicing-*

Is that satisfactory? Also, when should I mark UWP 5.3.2 as stable?